charts/super-agent/templates/_helm-release.yaml

```diff
@@ -0,0 +1,37 @@
+{{- define "newrelic-super-agent.helm.release" -}}
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  labels:
+    {{- include "newrelic.common.labels" . | nindent 4 }}
+  name: {{ include "newrelic.common.naming.fullname" . }}
```

Comment:
if `super-agent` to install the chart this name ends up also having `super-agent` which ends up in a really wired bug. 
I think that the common library internally removes the name duplication and this is why both names are the same.

Reply:
Solved on [#96cd608](https://github.com/newrelic/helm-charts/pull/1175/commits/96cd608894994e249a94c104964dd6834031b8d5)